### ISSUE TITLE
Drawable-Left image vanishes after clicking on edit text

### ISSUE THREAD

**Issue by himanshubabal** [scottyab/showhidepasswordedittext#17]
```
<com.scottyab.showhidepasswordedittext.ShowHidePasswordEditText
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:inputType="textPassword"
        android:id="@+id/password_editText"
        android:hint="Type your Password"
        android:drawableLeft="@drawable/key"
        android:drawablePadding="15dp" />
```

The Left-Drawable image vanishes after we start typing in the edit text

**Comment by scottyab** [comment]
Thanks for reporting, this could be related to our support of RTL, we should probably include a example of using left drawable in the sample app.

**Comment by scottyab** [comment]
@himanshubabal this is fixed and will be in 0.8 thanks again for raising it
